### ISSUE TITLE
Fix gulp run on travis

### ISSUE THREAD

**Issue by discordier** [contao/package-manager#4]
Travis builds currently fail with a strict violation.
```
/home/travis/build/contao/package-manager/gulpfile.js:3
const browserify    = require('browserify');
^^^^^
SyntaxError: Use of const in strict mode.
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Liftoff.handleArguments (/home/travis/build/contao/package-manager/node_modules/gulp/bin/gulp.js:116:3)
    at Liftoff.<anonymous> (/home/travis/build/contao/package-manager/node_modules/gulp/node_modules/liftoff/index.js:193:16)
    at module.exports (/home/travis/build/contao/package-manager/node_modules/gulp/node_modules/liftoff/node_modules/flagged-respawn/index.js:17:3)
    at Liftoff.<anonymous> (/home/travis/build/contao/package-manager/node_modules/gulp/node_modules/liftoff/index.js:185:9)
```

**Comment by psi-4ward** [comment]
Put a `"use strict";` at the beginning of your gulpfile?

**Comment by discordier** [comment]
Fixed in 5370af3875d2130142f0ca09550166357063554e
